src/ucs/config/parser.c

```diff
@@ -1144,7 +1144,7 @@ static ucs_status_t ucs_config_apply_env_vars(void *opts, ucs_config_field_t *fi
             if (ucs_config_is_deprecated_field(field)) {
                 if (added && !ignore_errors) {
                     ucs_warn("%s is deprecated (set %s%s=n to suppress this warning)",
-                             buf, UCS_CONFIG_PREFIX,
+                             buf, UCS_DEFAULT_ENV_PREFIX,
```

Comment:
need to print with relevant prefix

Reply:
you mean UCS_DEFAULT_ENV_PREFIX? It's fine I think, cause to disable deprecated vars warning we need to use UCX_WARN_UNUSED_ENV_VARS no matter what env prefix of variable is

Reply:
need to print the variable name the same way it was provided by user